core/plannernotes.c

```diff
@@ -418,14 +418,19 @@ void add_plan_to_notes(struct diveplan *diveplan, struct dive *dive, bool show_d
 					mingas_depth = get_depth_units(lastbottomdp->depth.mm, NULL, &depth_unit);
 					/* Print it to results */
 					if (cyl->start.mbar > lastbottomdp->minimum_gas.mbar) snprintf(mingas, sizeof(mingas),
-						translate("gettextFromC", "<br>&nbsp;&mdash; <span style='color: %s;'>Minimum gas</span> (based on %.1fxSAC/+%dmin@%.0f%s): \
-							%.0f%s/%.0f%s/<span style='color: %s;'>&Delta;:%+.0f%s</span>"),
+						"<br>&nbsp;&mdash; <span style='color: %s;'>%s</span> (%s %.1fx%s/+%d%s@%.0f%s): \
+							%.0f%s/%.0f%s<span style='color: %s;'>/&Delta;:%+.0f%s</span>",
```

Comment:
So this takes a translation away, yet the text still contains "Delta" which should be translated.
What am I missing?

Reply:
This is not "Delta" but "&Delta;" which is a html Greek character. This needs no translation. But it brings us to the question if the &Delta; symbol will be understood by all our international users.

Reply:
DUH, silly me... I think that should be understandable.
With that, LGTM
@atdotde - any concerns?